lms/static/scripts/frontend_apps/components/ErrorDisplay.tsx

```diff
@@ -112,7 +112,7 @@ export default function ErrorDisplay({
   const message = formatErrorMessage(error, /* prefix */ description);
 
   return (
-    <Scrollbox classes="LMS-Scrollbox">
+    <Scroll classes="LMS-Scrollbox">
```

Comment:
We'll want to convert this to Tailwind utility classes (almost done removing all of the legacy SASS-managed styles). Looks like all `.LMS-Scrollbox` does is disable borders. I don't believe `Scroll` _has_ borders so you may just be able to remove the class here.

Reply:
Changes applied